backend/src/models/Migration.js

```diff
@@ -0,0 +1,5 @@
+export default {
+  title: { type: 'string', primary: true, token: true },
+  description: { type: 'string' },
+  migratedAt: { type: 'string', isoDate: true, default: () => new Date().toISOString() },
+}
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-23T14:04:37Z" title="Thursday, January 23rd 2020, 3:04:37 pm +01:00">Jan 23, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,5 +1,5 @@
 export default {
   title: { type: 'string', primary: true, token: true },
   description: { type: 'string' },
-  timestamp: { type: 'number', unique: true },
+  createdAt: { type: 'string', isoDate: true, default: () => new Date().toISOString() },
```


Yeah, so this is certainly `migratedAt`.